subscription/controls.py

```diff
@@ -364,6 +374,9 @@ def chargeForCyVerse(stripe_api_key, stripe_token, priceToCharge, stripeDescript
                 PaymentControl.sendCyVerseSyncFailedEmail(termLabel, username, purchaseDate, transactionId, error)
 
             PaymentControl.sendCyVerseEmail(msg, purchaseId, termLabel, partnerObj, emailAddress, firstname, lastname, priceToCharge, institute, transactionId, expirationDate, cardLast4, other)
+        else:
+            #send email to tech team for an error in purchase (possible duplicate)
+            PaymentControl.sendCyVersePaymentErrorEmail(username, charge_amount, message)
```

Comment:
Covers the payment failure case. Does not send an email for the duplicate user case.